Uplink-CLI.md

```diff
@@ -2,7 +2,7 @@
 
 The Uplink CLI is our client-side application that supports interacting with
 the Storj network. This tutorial assumes you have an account on a running
-Satellite and uses a [test network](Test-network) Satellite by default.
+Satellite and uses a [test network](https://github.com/storj/storj/wiki/Test-network) Satellite by default.
```

Comment:
I am not sure if these links should be fixed like this. They actually work well on the respective wiki page: https://github.com/storj/storj/wiki/Uplink-CLI

The changes in this storj/doc repo are actually transferred to the wiki.